src/core/ack_tracker.c

```diff
@@ -96,6 +96,42 @@ QuicAckTrackerAddPacketNumber(
         !RangeUpdated;
 }
 
+BOOLEAN
+QuicAckTrackerDidHitReorderingThreshold(
+    _In_ QUIC_ACK_TRACKER* Tracker,
+    _In_ uint8_t ReorderingThreshold
+    )
+{
+    if (ReorderingThreshold == 0) {
+        return FALSE;
+    }
+
+    uint64_t LargestUnackedPacketNumber = QuicRangeGetMax(&Tracker->PacketNumbersToAck);
+    uint64_t LargestReported = 0; // The largest packet number that could be declared lost 
+    uint64_t SmallestUnreportedMissingPacketNumber; 
+
+    if (Tracker->LargestPacketNumberAcknowledged >= ReorderingThreshold) {
+        LargestReported = Tracker->LargestPacketNumberAcknowledged - ReorderingThreshold + 1;
+    }
+
+    if (LargestReported > LargestUnackedPacketNumber) {
```

Comment:
I think this should be put in the `if` statement above. It's impossible to be hit if we didn't update `LargestReported` to something other than zero.